scripts/common/rook.sh

```diff
@@ -246,3 +246,33 @@ function rook_operator_ready() {
     fi
     return 0
 }
+
+function rook_is_health_to_upgrade() {
+    log "Awaiting to check if rook-ceph ceph is health"
+    if ! $DIR/bin/kurl rook wait-for-health 600 ; then
+        kubectl -n rook-ceph exec deploy/rook-ceph-tools -- ceph status
+        logFail "Failed to detect Rook Ceph is not healthy"
+        return 1
+    fi
+
+    log "Checking Rook versions and replicas"
+    kubectl -n rook-ceph get deployment -l rook_cluster=rook-ceph -o jsonpath='{range .items[*]}{.metadata.name}{"  \treq/upd/avl: "}{.spec.replicas}{"/"}{.status.updatedReplicas}{"/"}{.status.readyReplicas}{"  \trook-version="}{.metadata.labels.rook-version}{"\n"}{end}'
+    local rook_versions=
+    rook_versions="$(kubectl -n rook-ceph get deployment -l rook_cluster=rook-ceph -o jsonpath='{range .items[*]}{"rook-version="}{.metadata.labels.rook-version}{"\n"}{end}' | sort | uniq)"
```

Comment:
Can we move this code to the go binary ? Maybe make it part of `wait-for-health` command line flag ?

Reply:
That is how we check in all places.
See the rook addon code to the upgrades

We might could move forward but the the bin would also need to return the result for we are able to check the output which seems add a lot of complexities unnecessary.